templates/workspaces/base/terraform/airlock/variables.tf

```diff
@@ -4,3 +4,4 @@ variable "ws_resource_group_name" {}
 variable "enable_local_debugging" {}
 variable "services_subnet_id" {}
 variable "short_workspace_id" {}
+variable "tre_resource_id" {}
```

Comment:
Same as above - pass in the map **tre_workspace_tags** instead of recreating it in this module.